examples/subscription_handler_example.py

```
@@ -0,0 +1,668 @@
+        `callback_timeout`: HTTP timeout for callback requests (default: 10.0)
+
+        `max_retries`: number of retry attempts for failed callbacks (default: 3)
+
+        `retry_delay`: seconds to wait between retries (default: 1.0)
+        """
+        self.keep_alive_interval = keep_alive_interval
+        self.callback_timeout = callback_timeout
+        self.max_retries = max_retries
+        self.retry_delay = retry_delay
+
+    def supports(self, request: Request, data: dict) -> bool:
```

Ruff (0.15.1)

[warning] Unused method argument: `request`

(ARG002)